diff --git a/Ding-Lab - m5231142/drlFX/src/main.py b/Ding-Lab - m5231142/drlFX/src/main.py
index 9a162f7a..21974f1d 100644
--- a/Ding-Lab - m5231142/drlFX/src/main.py	
+++ b/Ding-Lab - m5231142/drlFX/src/main.py	
@@ -18,7 +18,7 @@ const.MA_WIN2 = 80
 const.MA_WIN3 = 50
 const.MA_WIN4 = 30
 const.MA_WIN5 = 20
-const.LOG_DIR = './result_MA5/'
+const.LOG_DIR = './result_MA0/'
 const.LOG_PER_STEP = const.LOG_DIR + 'log_per_step_'
 const.LOG_PER_EP   = const.LOG_DIR + 'log_per_episode_'
 
@@ -148,11 +148,11 @@ class FXTrade(gym.core.Env):
     def step(self, action):
         reward = None
         cu = self._hist_list[self.cur_id] # current rate
-        ma1 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN1:self.cur_id])
-        ma2 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN2:self.cur_id])
-        ma3 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN3:self.cur_id])
-        ma4 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN4:self.cur_id])
-        ma5 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN5:self.cur_id])
+        ma1 = 0
+        ma2 = 0
+        ma3 = 0
+        ma4 = 0
+        ma5 = 0
         fpl = 0 if self._trading.calc_pl(cu) == None else self._trading.calc_pl(cu)/const.SCALING
 
         self.cur_id +=1
@@ -214,11 +214,11 @@ class FXTrade(gym.core.Env):
     def reset(self):
         self.reset_FXTrade()
         self._trading.reset()
-        ma1 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN1:self.cur_id])
-        ma2 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN2:self.cur_id])
-        ma3 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN3:self.cur_id])
-        ma4 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN4:self.cur_id])
-        ma5 = stat.mean(self._hist_list[self.cur_id-const.MA_WIN5:self.cur_id])
+        ma1 = 0
+        ma2 = 0
+        ma3 = 0
+        ma4 = 0
+        ma5 = 0
         # TODO: # Think about validity of setting　0.0 on 5th element (pos_rate) as initial value
         return np.array([self._hist_list[self.cur_id], self._trading._position, ma1, ma2, ma3, ma4, ma5, 0.0, 0.0])
     
